### PR TITLE
Remove bad link

### PR DESCRIPTION
**Pull request by jstallm**
## Summary

Remove bad link

### CODE REVIEW

File: docs/standard/microservices-architecture/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md [hunk 1]
```diff
@@ -93,9 +93,6 @@ public class PrimeWebDefaultRequestShould
 -   **Steve Smith. Testing controllers** (ASP.NET Core)
     [*https://docs.microsoft.com/aspnet/core/mvc/controllers/testing*](/aspnet/core/mvc/controllers/testing)
 
--   **Steve Smith. Integration testing** (ASP.NET Core)
-    [*https://docs.microsoft.com/aspnet/core/testing/integration-testing*](/aspnet/core/testing/integration-testing)
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md]: The correct link to the topic is /aspnet/core/test/integration-tests

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for contributing to the dotnet/docs repo and for reporting this broken link, @jstallm. However, the link is merely incorrect; the topic does exist. Would you mind replacing the incorrect link with the correct one?

**Comment by rpetrusha** [comment]
//cc @CESARDELATORRE

**Comment by mairaw** [comment]
@rpetrusha I've fixed this for @jstallm.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**